plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/StatisticsCollectionConfirmDialog.java

```diff
@@ -59,20 +63,23 @@ protected Composite createDialogArea(@NotNull Composite parent) {
                         null,
                         PrefPageUsageStatistics.PAGE_ID);
                 }));
-
+        UIUtils.createEmptyLabel(composite, 1,1);
+        shareData = UIUtils.createCheckbox(composite, "Ask not to share", false);
```

Comment:
This sentence sounds a bit weird. Was it discussed prior to this change?

Something like "I do not want to share anonymous statistics" or "Don't share" would fit better, but it's up for discussion.

Reply:
Agreed

Reply:
No objections from my side, let's discuss with Anna.

Reply:
Ok, I agree. Please change it to "I don't want to share the data".
I hope it looks better.

Reply:
Fixed

Reply:
Please update the screenshot as well

Reply:
sorry guys, we have to do it again =) please change the checkbox label to **Do not share data**

Reply:
![image](https://github.com/dbeaver/dbeaver/assets/32031248/eddf0bfd-423b-4cf1-a9c1-117c966d1699)

Reply:
![image](https://github.com/dbeaver/dbeaver/assets/32031248/175ac1e6-edb7-4fa0-970d-844cc082f0fb)